examples/brand/app.py

```diff
@@ -61,13 +64,15 @@
         ui.layout_column_wrap(
             ui.card(
                 ui.card_header("Button Variants"),
-                ui.input_action_button("btn1", "Default"),
-                ui.input_action_button("btn2", "Primary", class_="btn-primary"),
-                ui.input_action_button("btn3", "Secondary", class_="btn-secondary"),
-                ui.input_action_button("btn4", "Info", class_="btn-info"),
-                ui.input_action_button("btn5", "Success", class_="btn-success"),
-                ui.input_action_button("btn6", "Warning", class_="btn-warning"),
-                ui.input_action_button("btn7", "Danger", class_="btn-danger"),
+                ui.input_action_button("default_btn", "Default"),
```

Comment:
This is very much a minor nit, but would you mind if we used IDs like `btn_*`, `value_box_*`, etc? E.g. `btn_default`, `btn_primary`, `value_box_primary`, etc.

Reply:
Sure, made the change it the latest commit.